sumpy/expansion/local.py

```diff
@@ -348,6 +381,7 @@ def calc_max_mi(mis):
             # to compensate for differentiating which is done at the end.
             # This moves the two cancelling "rscales" closer to each other at
             # the end in the hope of helping rscale magnitude.
+            from sumpy.tools import MiDerivativeTaker
```

Comment:
```suggestion
```

Reply:
This was imported early in the function before and it's moved to where it's used now